install-tbtc.sh

```diff
@@ -9,35 +9,50 @@ DONE_END='\n\n\e[0m'    # new line + reset
 
 WORKDIR=$PWD
 
+BTC_NETWORK="regtest"
+
+while getopts n: option; do
+  case "${option}" in
+  n) BTC_NETWORK=${OPTARG};;
+  *) echo "invalid option";;
+  esac
+done
+
 printf "${LOG_START}Starting tBTC deployment...${LOG_END}"
 
 cd "$WORKDIR/relay-genesis"
 npm install
 
-cd "$WORKDIR/tbtc/solidity/migrations"
+printf "${LOG_START}Using $BTC_NETWORK Bitcoin network${LOG_END}"
 
-# Always deploy TestnetRelay instead of the defaull MockRelay.
-jq --arg forceRelay TestnetRelay '. + {forceRelay: $forceRelay}' relay-config.json > relay-config.json.tmp && mv relay-config.json.tmp relay-config.json
+if [ "$BTC_NETWORK" == "testnet" ]; then
+  cd "$WORKDIR/tbtc/solidity/migrations"
```

Comment:
```suggestion
  # Deploy TestnetRelay instead of the defaull MockRelay.
  printf "${LOG_START}Updating relay configuration...${LOG_END}"
  cd "$WORKDIR/tbtc/solidity/migrations"
```